products/identity/Policies and Procedures/Require_MFA_VAgov.md

```diff
@@ -0,0 +1,38 @@
+# MFA Required on VA.gov
+
+### Version History
+| Version Number | Author(s) | Revision Date | Description of Change |
+| --- | --- | --- | --- |
+| 1.0 | Joe Niquette | 5/19/23 | Initial creation, approval provided via email to the VA.gov Identity team by Cody Reinold. This document formally acknowledges that approval by the PR approval of Thomas Black. |
+
+## Background
+
+Fraudulent Direct Deposit account changes on VA.gov are on the rise at VA. In an attempt to reduce fraudulent activity on VA.gov the VA.gov Identity team has implemented a policy that requires all direct deposit changes be made by an MFA (multi-factor authentication) required verified ID.me or Login.gov account. This policy was set by the Identity teams PO (Product Owner) at the time the policy was set, Cody Reinold. The policy was set and is not set to expire effective 9/20/2021.
```

Comment:
> Fraudulent Direct Deposit account changes on VA.gov are on the rise at VA. In an attempt to reduce fraudulent activity on VA.gov the VA.gov Identity team has implemented a policy that requires all direct deposit changes be made by an MFA (multi-factor authentication) required verified ID.me or Login.gov account. This policy was set by the Identity teams PO (Product Owner) at the time the policy was set, Cody Reinold. The policy was set and is not set to expire effective 9/20/2021.

Fraudulent direct deposit account changes on VA.gov are on the rise at the Department of Veterans Affairs. In an attempt to reduce fraudulent activity on VA.gov, the VA.gov Identity team has implemented a policy that requires all direct deposit changes to be made with a verified ID.me or Login.gov account that requires MFA (multi-factor authentication). This policy was set by the Identity team's PO (Product Owner) at the time, Cody Reinold. The policy has been set and is not set to expire until 9/20/2021.